protobuf/mayastor.proto

```diff
@@ -500,6 +503,35 @@ message RebuildStatsReply {
   uint64 tasks_active = 7; // number of current active tasks
 }
 
+enum RebuildJobState {
+  INIT = 0;        // Initialized, not yet rebuilding
+  REBUILDING = 1;  // Running rebuild
+  STOPPED = 2;     // Stopped rebuild
+  PAUSED = 3;      // Paused rebuild
+  FAILED = 4;      // Failed for some reason
+  COMPLETED = 5;   // Rebuild completed succesfully
+}
+
+message RebuildHistoryRequest {
+  string uuid = 1;  // uuid of the nexus
+}
+
+message RebuildHistoryReply {
+  string nexus = 1;                          // uuid of the nexus
+  repeated RebuildHistoryRecord records = 2; // List of all rebuild records
+                                             // across all the replicas
+}
+
+message RebuildHistoryRecord {
+  string dst_uri = 1;            // URI of rebuilt replica (destination)
+  string src_uri = 2;            // URI of source replica
+  RebuildJobState state = 3;     // Final state of rebuild job
+  uint64 rebuilt_data_size = 4;  // Size of data rebuilt
+  bool   is_partial = 5;         // Full or Partial rebuild
+  google.protobuf.Timestamp  started_at = 6;        // Start timestamp(UTC) of rebuild job
```

Comment:
Is the double space between Timestamp and started_at/ended_at intended?

Reply:
not intended, fixed.